pkg/proto/configuration/blobstore/blobstore.proto

```diff
@@ -308,6 +308,70 @@ message MirroredBlobAccessConfiguration {
   BlobReplicatorConfiguration replicator_b_to_a = 4;
 }
 
+// LocalBlobAccess stores all data onto disk in block sizes. A block
+// cannot span multiple blocks, meaning that blocks generally need to
+// be large in size (gigabytes). The number of blocks may be relatively
+// low. For example, for a 512 GiB cache, it is acceptable to create 32
+// blocks of 16 GiB in size.
+//
+// Blocks are partitioned into three groups based on their creation
+// time, named "old", "current" and "new". Blobs provided to Put() will
+// always be stored in a block in the "new" group. When the oldest block
+// in the "new" group becomes full, it is moved to the "current" group.
+// This causes the oldest block in the "current" group to be displaced
+// to the "old" group. The oldest block in the "old" group is discarded.
+//
+// The difference between the "current" group and the "old" group is
+// that the needRefresh value returned by Get() differs.
```

Comment:
I think that for OldCurrentNewLocationBlobmap this phrasing makes sense, because we want to document how it interacts with other classes. For blobstore.proto I think it would make more sense to just explain the implications of this from a user's perspective:

```
// The difference between the "current" group and the "old" group is
// that data is refreshed when accessed.
```